SPECS/cloud-init/cloud-init.spec

```diff
@@ -1,16 +1,17 @@
+%define cl_services cloud-config.service cloud-config.target cloud-final.service cloud-init.service cloud-init.target cloud-init-local.service
 Summary:        Cloud instance init scripts
 Name:           cloud-init
-Version:        22.2
```

Comment:
I think this is a merge bug - we don't want to roll back a spec update. And I'm guessing this change isn't about specs at all.;)